sources/packages/backend/apps/api/src/route-controllers/cas-supplier/_tests_/cas-supplier.aest.controller.getCASSupplier.e2e-spec.ts

```diff
@@ -0,0 +1,96 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  saveFakeCASSupplier,
+} from "@sims/test-utils";
+import { CASSupplierSiteStatus, SupplierStatus } from "@sims/sims-db";
+
+describe("CASSupplierAESTController(e2e)-getCASSuppliers", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it("Should get all the CAS suppliers for a student when requested given a student id.", async () => {
+    // Arrange
+    const now = new Date();
+    const fakeCASSupplierInfo1 = {
+      supplierStatus: SupplierStatus.Verified,
```

Comment:
Would the intention here be to validate the `SupplierStatus.VerifiedManually` which will have limited information to be retrieved?

Reply:
The intention is just to validate that the retrieved values are the same from what there is in the table independent of what is there. If you would like to have a record with the minimum number of columns populated like for the case of SupplierStatus.VerifiedManually I can add that one too.

Reply:
I do not believe that we should add one more case but I would like to have one case where the minimum data is retrieved and validated. Right now I believe the `SupplierStatus.VerifiedManually` would be the one to try to replicate.